src/components/home/employees/EmployeeFilter.js

```diff
@@ -0,0 +1,43 @@
+import React, { Component } from 'react';
+import debounce from 'lodash/debounce';
+
+const FILTER_SEARCH_DELAY = 300;
```

Comment:
Lets name it `SEARCH_DELAY`
```suggestion
const SEARCH_DELAY = 300;
```

Reply:
@pratishshr  Should I define this in constants file too? Maybe `constants/configuration`?

Reply:
No, let's keep it here for now, since it's only being used in this component.